person.rb

```diff
@@ -1,8 +1,15 @@
-class Person
+class Namable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] I suggest you explicitly tell the user why this error was raised and what the problem is. This will make sure that the user doesn't get frustrated and thinking what could be the error.

```suggestion
  def correct_name
    raise NotImplementedError("Nameable has not implemented correct_name method")
  end
```